Assignment 5.Rmd

```diff
@@ -75,3 +76,14 @@ D2$prediction <- predict(score_ctree, D2)
 ``` 
 ## Part III
 Compare the predicted advice with the actual advice that these students recieved. What is the difference between the observed and predicted results?
+```{r}
+D2$advice <- ifelse(D2$score <=0.4, "intervene", ifelse(D2$score > 0.4 & D2$score <=0.8, "monitor", "no action"))
+D2$match <- ifelse(D2$prediction == D2$advice, "Yes", "No")
+sum(D2$match == "Yes")
+#The total number of students who match the actual and predicted results is 116
+sum(D2$match == "No")
```

Comment:
This seems like a very high number. Are you sure? You could achieve the result with the following code: `mean(ifelse(D2$prediction == "no action", 1, 0))`